composer.json

```diff
@@ -17,7 +17,11 @@
         "phpspec/prophecy": "^1.6",
 
         "doctrine/orm": "^2.5",
-        "doctrine/dbal": "^2.5"
+        "doctrine/dbal": "^2.5",
+
+        "symfony/dependency-injection": "^2.8@dev|^3.0@dev",
```

Comment:
I'm not sure about those `@dev` dependencies, but if we would like to keep `"minimum-stability": "stable"` I think there is no other way.